neuralop/datasets/data_transforms.py

```diff
@@ -108,6 +114,12 @@ def forward(self, **data_dict):
         output = self.model(data_dict["x"])
         output = self.postprocess(output)
         return output, data_dict
+    
+    def train(self):
+        self._train = True
+    
+    def train(self):
+        self._train = False
```

Comment:
I don't get these two functions - you're redefining? 
We should define `train` as a property. But if it's an nn.Module why not reuse the existing one?

Reply:
Good idea